lib/chef/http/authenticator.rb

```diff
@@ -215,14 +254,46 @@ def self.decrypt_pfx_pass(password)
         powershell_exec!(powershell_code).result
       end
 
-      def self.retrieve_certificate_key(client_name)
-        require "openssl" unless defined?(OpenSSL)
+      def self.migrate_pass_to_use_vector(password)
+        Chef::Config[:auth_key_registry_type] == "user" ? store = "HKCU" : store = "HKLM"
```

Comment:
nit: I feel like we do this in a few places, should this be a helper function?

Reply:
done. There were two of them. One for the certificate store and one for the registry store.